.github/workflows/test-beta.yml

```
@@ -0,0 +1,25 @@
+name: test-beta
+
+on:
+  schedule:
+    - cron: "0 6 * * *"
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@main
```

Let's run the tests against the latest dev work

```suggestion
      - uses: actions/checkout@next
```